x-pack/plugin/gpu/src/yamlRestTest/java/org/elasticsearch/xpack/gpu/GPUClientYamlTestSuiteIT.java

```diff
@@ -35,7 +35,8 @@ private static ElasticsearchCluster createCluster() {
             // Needed to get access to raw vectors from Lucene scorers
             .jvmArg("--add-opens=org.apache.lucene.core/org.apache.lucene.codecs.lucene99=org.elasticsearch.server")
             .jvmArg("--add-opens=org.apache.lucene.core/org.apache.lucene.codecs.hnsw=org.elasticsearch.server")
-            .jvmArg("--add-opens=org.apache.lucene.core/org.apache.lucene.internal.vectorization=org.elasticsearch.server");
+            .jvmArg("--add-opens=org.apache.lucene.core/org.apache.lucene.internal.vectorization=org.elasticsearch.server")
+            .jvmArg("-Des.gpu_vectors_indexing_feature_flag_enabled=true");
```

Comment:
I am not 100% sure this is how we want to do it. I think we need to apply it in `rest-api-spec/src/yamlRestTest/java/org/elasticsearch/test/rest/ClientYamlTestSuiteIT.java` or the class directly, there is `public static ElasticsearchCluster cluster = ElasticsearchCluster.local()` node construction, there you can add `.feature(FeatureFlag....`

See an example of removal here: https://github.com/elastic/elasticsearch/pull/135299/files